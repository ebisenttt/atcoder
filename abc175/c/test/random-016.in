-105897566724068 659524340221778 359520104419826
