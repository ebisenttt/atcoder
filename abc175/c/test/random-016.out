105897566724068
